src/main/java/oshi/software/os/linux/LinuxFileSystem.java

```diff
@@ -63,8 +64,8 @@
                 if (split.length < 6) {
                     continue;
                 }
-                fstype.put(split[1].replaceAll("\\\\040", " "), split[2]);
-            }
+                    fstype.put(split[1].replaceAll("\\\\040", " "), split[2]);
+                }

```

Comment:
Mmm... It's strange, I don't remember that I've touched this part of code (maybe I've reformatted automatically and I dont' remember). However, I use Netbeans with default settings for code formatting